src/lib/audienceChannel/styles/slides.css

```diff
@@ -0,0 +1,50 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font: 13px Helvetica, Arial;
+}
+
+#present {
+  width: 800px;
+  height: 600px;
+  margin: 0 auto;
+  margin-top: 0px
+}
+
+#previous {
+  background-image: url(/lib/img/previous.png);
+  display: inline-block;
+  width: 44px;
+  height: 44px;
+  outline: none;
+  text-indent: -9999px;
+  margin-top: 0px;
+}
+
+#previous:hover {
+  background-position: 0 -44px;
+}
+
+#next {
```

Comment:
next button is used by both slides.html and phresenter.html for question dialog, can it be put into a common css?